lib/classes/config-schema-handler/index.js

```diff
@@ -53,6 +53,9 @@ const normalizeUserConfig = (userConfig) => {
 };
 const denormalizeUserConfig = (userConfig, { removedValuesMap }) => {
   for (const removedValueData of removedValuesMap) {
+    // #7747, cannot remove _.set here, as paths in `removedValueData.path` are complex
+    // and deep, though these paths in `userConfig` always seem to exist during testing
+    // this thread may be useful in future: https://stackoverflow.com/q/6491463/5343977
```

Comment:
Advisory was just to address cases where `_.get` is used to assign properties that can be replaced with `obj.prop = value` (as when issue was created, some of such usage was spread).

All other cases need to be left intact.

So let's leave this case, and remove that comment

Reply:
Will do.